index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+  <head>
+    <meta charset="utf-8">
+    <link rel="stylesheet" href="styles.css">
+    <script src="script.js" defer></script>
+    <title> Welcome! </title>
+  </head>
+  <body>
+    <h1>My Introduction &#128123; </h1>
+    <table style="width:50%">
+      <tr>
+        <th></p><strong>I'm Raynelle </strong> </p>
+          Current Location: London <img src="https://img.icons8.com/color/48/000000/great-britain.png" width="20" height="20"></th>
+        <th></th> 
```

Comment:
Any special reason for these extra markups?

Reply:
Ohh we don't need these..Removed